controllers/childGuidance.js

```diff
@@ -0,0 +1,133 @@
+import childModel from '../models/childModel.js';
+
+const createGuidance = async (req, res) => {
+  try {
+    const data = req.body;
+    const check = await childModel.findOne({ guideId: data.guideId });
+    if (check) {
+      return res.status(409).json({
+        message: "Child guide already exists",
+      });
+    }
+
+    const guideInstance = new childModel({
+      name: data.name,
+      age: data.age,
+      guides: [
+        {
+          question: "My child has a fever, what can I do?",
+          answer:
+            "If your child has a fever, you have to visit the closest hospital as soon as possible",
+          category: "Healthcare",
+        },
+      ],
+      guideId: data.guideId,
+    });
+
+    const result = await guideInstance.save();
+
+    res.status(200).json({
+      message: "Child Guidance created successfully",
+      error: null,
+      data: result,
+    });
+  } catch (error) {
+    res.status(500).json({
+      message: "Error in the createGuidance controller",
+      error: error,
+      data: null,
+    });
+  }
+}
+
+
+
+
+const getGuidance = async (req, res) => {
+  try {
+    const guideId = req.params.guideId;
+    const check = await childModel.findOne({ guideId: guideId });
+    if (!check) {
+      return res.status(409).json({
+        message: "The child Guide you are trying to read is not available",
+        error: "Error fetching the Child Guide",
+        data: null,
+      });
+    }
+    res.status(200).json({
+      message: "The child Guide fetched successfully",
+      error: null,
+      data: check,
+    });
+  } catch (error) {
+    res.status(500).json({
+      message: "Error in the getGuidance controller",
+      error: error,
+      data: null,
+    });
+  }
+};
```

Comment:
- this controller for getting one guide is fine, please also work on another endpoint for fetching all guides created